New package: ThickIntervals v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: ThickIntervals
- Repository: https://github.com/lorenzgillner/ThickIntervals.jl
- Created by: @lorenzgillner
- Version: v0.1.0
- Commit: 4352ab8e6d9a6a56b6c70938846650785df66bda
- Git reference: HEAD

goerz commented:
Thank you for submitting your package! Could you add a little bit of documentation before registering? [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README. An important part of packages in General is that any potential user can figure out what the package is about and how to get started with using it. That is really difficult when there is no documentation.

In the longer term, I definitely recommend setting up a [Documenter](https://documenter.juliadocs.org/stable/)-based documentation. Before a `v1.0` release, or for smaller packages that can be effectively described entirely with their `README`, that's not a requirement, though.

JuliaHubRegistrator committed:
New package: ThickIntervals v0.1.0

UUID: 540b2d50-f826-4de2-88b6-7b05931975d3
Repo: https://github.com/lorenzgillner/ThickIntervals.jl.git
Tree: 3216aa932366b20d1fb2d966371b7c3ea0cfadc8

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
